src/components/specific/models/models-manager/models-table/ModelsTable.vue

```diff
@@ -91,11 +122,23 @@ export default {
       { immediate: true }
     );
 
+    const background = computed(
+      () =>
+        `var(--color-silver-light) url("/static/modelsManager/menuAnimation/${
```

Comment:
You can use `$i18n.availableLocales` (I don't know how to to it in vue3) to automatically get them

Reply:
Apparently it's
```javascript
const  {$i18n } = useI18n();
$i18n.availableLocales
``

Reply:
I did not use i18n locale variable here because we do not translate every locale, but only 5 of them (FR, DE, EN, IT, ES)
Therefore, I made a specific list